project/plugins.sbt

```
@@ -1,4 +1,4 @@
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.13.0")
-addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.10.0")
-addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "0.7.4")
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.2")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.14.3")
+addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.11.0")
+addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "0.7.7")
```

we should problably go all the way to 0.8.0, since I have moved to central.
https://github.com/typelevel/sbt-typelevel/releases/tag/v0.8.0

```suggestion
addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "0.8.0")
```